src/views/hoc/hoc.scss

```diff
@@ -8,123 +8,117 @@ $base-bg: $ui-white;
 }
 
 .hoc {
-    .title-banner {
-        &.wbb-bg {
-            background-image: url("/images/hoc2015/hide-bg.jpg");
+    .img,
+    .button {
+        &.card-info {
+            width: calc(100% - 20px);
         }
+    }
 
-        &.dance-bg {
-            background-image: url("/images/hoc2015/dance-bg.jpg");
+    .img {
+        &.card-info {
+            margin: 10px 10px 5px 10px;
+            border-radius: 5px;
         }
 
-        &.name-bg {
-            background-image: url("/images/hoc2015/name-bg.jpg");
+        &.mod-border {
```

Comment:
The image stylings on this page do not currently align with image stylings elsewhere. Consequently, I'd prefer to wait on thinking about an `Image` component until we re-factor elsewhere and start to see the necessity for it.